cluster.go

```diff
@@ -82,15 +84,26 @@ type clusterClient struct {
 	aws    bool
 }
 
+// NOTE: clusterConnection and conn must be initialized at the same time
+type clusterConnection struct {
+	conn              conn
+	isPrimaryNodeConn bool
```

Comment:
```suggestion
	replica bool
```
I think it is better to consider a conn not a replica connection by default whereas the current isPrimaryNodeConn treats a conn not a primary connection by default.

Reply:
Also changed it too. [3bb4fb5](https://github.com/redis/rueidis/pull/402/commits/3bb4fb58e43879fa87824249b8325a4751323c7d)